meudesafio/page.html

```diff
@@ -139,21 +139,23 @@ <h2 class="skills">Habilidades</h2>
         <div class="empty"></div>
         <div class="fill">Wordpress</div>
       </div>
-      <div class="contact-wrapper">
-        <h1 class="section-title">Contato</h1>
-        <span class="text black">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus vitae ullamcorper
-          est.</span>
-        <span class="email">meuemail@mail.com</span>
-        <span class="telefone">+55 21 997115009</span>
-        <h3 class="midia-title">Midias</h3>
-        <div class="social-media">
-          <i class="facebook">F</i>
-          <i class="Twitter">T</i>
-          <i class="LinkedIn">LI</i>
-          <i class="Pinterest">P</i>
-          <i class="Ball">O</i>
-          <i class="Google">G</i>
-        </div>
-
+    </section>
+    <sectsion class="contact-wrapper">
+      <h1 class="section-title">Contato</h1>
+      <span class="text black">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus vitae ullamcorper
+        est.</span>
+      <span class="email">meuemail@mail.com</span>
+      <span class="telefone">+55 21 997115009</span>
+      <h3 class="midia-title">Midias</h3>
+      <div class="social-media">
+        <i class="facebook">F</i>
+        <i class="Twitter">T</i>
+        <i class="LinkedIn">LI</i>
+        <i class="Pinterest">P</i>
+        <i class="Ball">O</i>
+        <i class="Google">G</i>
       </div>
-      <footer>Woman + Tech = < /strong> </footer>
\ No newline at end of file
+  </section>
+
+  </div>
+  <footer>Woman + Tech = < /strong> </footer> </section>
```

Comment:
@marianaruddy acho que essa tá perdida... onde ela abre?

```suggestion
    <section>
        <footer>Woman + Tech = </strong> </footer>
    </section>
```